### DISCUSSION TITLE
Mac OS install issue

### DISCUSSION THREAD

**Discussion by MAJINFORMATIQUE** [amidaware/tacticalrmm#1306] [General]
**Server Info (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04, Debian 10] : 
 - Browser: [e.g. chrome, safari]
 - RMM Version (as shown in top left of web UI): 0.15

**Installation Method:**
  - [x] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent OS: Mac OSx

**Describe the bug**
Impossible to install mesh agent FATA [0031].
Test on a macbook pro 2017 core i5 16 Go RAM
Using code signing

**Comment by dinger1986** [comment]
can you add a screenshot of the issue? I tested it as soon as it was available and it worked fine

**Comment by wh1te909** [comment]
What version of osx?

**Reply by MAJINFORMATIQUE** [reply]
11.6 and 12.6. I've resolved partially the issue : download from mesh web console the meshagent.mpkg. Deploy on mac. Add Security acces disk and record on  Mac. Copy install link from tactical rmm with option -nomesh. Then installation ok on tactical RMM but Mesh agent run from mesh web console only. On tactical RMM web console white screen when take control.
